Lib/asyncio/unix_events.py

```diff
@@ -680,7 +646,7 @@ def _start(self, args, shell, stdin, stdout, stderr, bufsize, **kwargs):
             # needed by close_fds=False on Python 3.3 and older
             # (Python 3.4 implements the PEP 446, socketpair returns
             # non-inheritable sockets)
-            _set_inheritable(stdin_w.fileno(), False)
+            os.set_inheritable(stdin_w.fileno(), False)
```

Comment:
Does not the above comment make this unneeded? @vstinner, what are your thoughts?